quic-encryption-offload.md

```diff
@@ -201,6 +211,8 @@ Value | Meaning
 
 If the `QEO_DECRYPTION_STATUS` ancillary data is not present then there was no offloaded connection that matched the QUIC packet(s).
 
+The payload of a decryption failure is not the original payload sent on the wire, but the result of the failed decryption.
```

Comment:
Should the text capture the requirement from [#32](https://github.com/microsoft/quic-offloads/issues/32)? Its better to call this out if it is needed.

> If the client doesn't use a CID, and the received packet is a stateless reset. In this case we'd still need at least the original 16 bytes of what could be the stateless reset token passed up.

Reply:
A failed decryption attempt shouldn't modify the tag at the end (or at least what you think is the tag) right? Generally, the process would be to **read** those last 16 bytes and compare to the result of what you computed during the decrypt operation. If I'm mistaken, then I add text to ensure the last 16 bytes aren't modified.

Reply:
That should be an OK assumption in that a failed decryption attempt shouldn't have modified the tag at the end. But, from a spec perspective, if the process needs to read those last 16 bytes of the original payload (i.e. the tag), then it is better to explicitly call out this out in the spec. It makes the "contract" clear for a HW implementation and helps with validation use cases etc.

Reply:
Ok. I will explicitly state this in the text then. Thanks for the feedback!